README.md

```diff
@@ -47,18 +49,6 @@ repository, and the results are hosted on the
 <br>
 
 
-## Index
-
-* [Developer environment](https://github.com/conbench/conbench#developer-environment)
-* [Configuring the server](https://github.com/conbench/conbench#configuring-the-server)
-* [Creating accounts](https://github.com/conbench/conbench#creating-accounts)
-* [Authoring benchmarks](https://github.com/conbench/conbench#authoring-benchmarks)
-  * [Simple benchmarks](https://github.com/conbench/conbench#example-simple-benchmarks)
-  * [External benchmarks](https://github.com/conbench/conbench#example-external-benchmarks)
-  * [Case benchmarks](https://github.com/conbench/conbench#example-case-benchmarks)
-  * [R benchmarks](https://github.com/conbench/conbench#example-r-benchmarks)
```

Comment:
These were already out-of-date, and the new site has everything on the left sidebar.